### DISCUSSION TITLE
Welcome to WilderWild Discussions!

### DISCUSSION THREAD

**Discussion by Soro335** [FrozenBlock/WilderWild#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EthanRStokes** [comment]
hi, im treetrain1
i do a lot of things with the community 😏 
ronald mcdonald stop hiding in my walls ronald mcdonald stop hiding in my walls ronald mcdonald stop hiding in my walls ronald mcdonald stop hiding in my walls ronald mcdonald stop hiding in my walls

**Comment by elmur0d11** [comment]
Hello bro you repositorys is just bomb!! 
And I have one small question > how i can found ideas to create something like API's or applicatations what do yo think?
